common/types_test.go

```diff
@@ -346,21 +346,21 @@ func TestAddress_Value(t *testing.T) {
 		0xa2, 0x18, 0xc6, 0xa9, 0x27, 0x4d, 0x30, 0xab, 0x9a, 0x15,
 	}
 	var usedA Address
-	usedA.inner.setBytes(b)
+	usedA = BytesToAddress(b);
 	tests := []struct {
 		name    string
 		a       Address
 		want    driver.Value
 		wantErr bool
-	}{
-		{
-			name:    "Working value",
-			a:       usedA,
-			want:    b,
-			wantErr: false,
-		},
-	}
-	for _, tt := range tests {
+		}{
```

Comment:
I think the indentation should be left as it was

Reply:
fixed